hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/crypto/key/TestValueQueue.java

```diff
@@ -111,6 +119,40 @@ public void testWarmUp() throws Exception {
     vq.shutdown();
   }
 
+  /**
```

Comment:
[ERROR] /Users/sdzinama/dev/hadooptree/trunk/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/crypto/key/TestValueQueue.java:122: First sentence should end with a period. [JavadocStyle]